inscriptions/ISic020919.xml

```diff
@@ -0,0 +1,227 @@
+                            <origPlace>
+                                <placeName type="ancient" ref="http://pleiades.stoa.org/places/462197">Entella</placeName>
+                                <placeName type="modern" ref="http://sws.geonames.org/11103525/">Rocca d'Entella</placeName>
+                                <geo>37.77889,13.11509</geo>
+                            </origPlace>
+                            <origDate datingMethod="#julian" evidence="material-context lettering" notBefore-custom="-0500" notAfter-custom="-0450">500—450 BCE (?)</origDate>
+                        </origin>
+                        <provenance type="found">Sporadic find. From the 'central area' (Ampolo 2019) of the settlement of Entella<geo>37.77889,13.11509</geo>.</provenance><!-- generic coordinates for Entella -->
+                        <acquisition/>
+                    </history>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>Encoded following the latest EpiDoc guidelines</p>
+            <xi:include href="../alists/ISicily-taxonomies.xml">
+                <xi:fallback>
+                    <p>Taxonomies for ISicily controlled values</p>
+                </xi:fallback>
+            </xi:include>
+            <xi:include href="../alists/charDecl.xml">
+                <xi:fallback>
+                    <p>ISicily glyphs authority list</p>
+                </xi:fallback>
+            </xi:include>
+        </encodingDesc>
+        <profileDesc>
+            <calendarDesc>
+                <calendar xml:id="julian">
+                    <p>Julian Calendar</p>
+                </calendar>
+            </calendarDesc>
+            <langUsage>
+                <language ident="en">English</language> 
+                <language ident="it">Italian</language> 
+                <language ident="grc">Ancient Greek</language> 
+                <language ident="la">Latin</language> 
+                <language ident="he">Hebrew</language> 
+                <language ident="phn">Phoenician</language>
+                <language ident="xpu">Punic</language>
+                <language ident="osc">Oscan</language> 
+                <language ident="xly">Elymian</language> 
+                <language ident="scx">Sikel</language>  
+                <language ident="sxc">Sikan</language>  
+            </langUsage>
+            <textClass>
+                <keywords scheme="http://www.eagle-network.eu/voc/typeins.html">
+                    <term ana="#function.unknown" ref="https://www.eagle-network.eu/voc/typeins/lod/143.html">unknown</term>
+                </keywords>
+            </textClass>
+        </profileDesc>
+        <revisionDesc status="draft">
+            <listChange>
+                <change when="2020-10-05" who="#SS">Simona Stoyanova normalised Unicode</change>
+                <change when="2020-10-08" who="#SS">Simona Stoyanova updated list of languages</change>
```

Comment:
for a new file created now, there shouldn't be change notes that predate the file's creation!